Multisig/UI/UI Library/Views/AddressInfoView.swift

```diff
@@ -55,6 +55,7 @@ class AddressInfoView: UINibView {
         if showIdenticon {
             identiconView.setCircleImage(url: imageUri, address: address)
         }
+        identiconView.isHidden = !showIdenticon
```

Comment:
is it an existing bug fix?

Reply:
when I hidden the address button, it turned out that the address was not centered because the identiconview hadn't any image and was used in the layout. So I hide it here.